app/Console/Commands/PollTwitter.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\User;
+use Illuminate\Console\Command;
+use Log;
+
+class PollTwitter extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'suggestive:poll-twitter';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Polls Twitter for any tweets with the configured hashtag.';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        $users = User::where('role', 'owner')
+                     ->whereNotNull('hashtag')
+                     ->get([ 'id', 'name', 'hashtag', 'last_tweet_id', ]);
+
+        if ($users) {
+            $twitter = app('TwitterClient');
```

Comment:
It's not an interface, it's an (I don't know what the proper name is) alias to the underlying return, given the `TwitterOAuth` client doesn't implement an interface.

Would you suggest that in this instance we would create an app-specific wrapper for it that implements an interface so that we can bind to it?

This particular solution has always seemed hacky to me (I use the same method in another app I have). The alternative would be to wrap it in a facade?